openfisca_uk/variables/finance/benefit/family/universal_credit.py

```diff
@@ -568,3 +579,50 @@ def formula(benunit, period, parameters):
             * benunit("claims_UC", period)
             * benunit("is_UC_eligible", period)
         )
+
+
+class UC_MIF_applies(Variable):
+    value_type = bool
+    entity = Person
+    label = "Minimum Income Floor applies"
+    documentation = "Whether the Minimum Income Floor should be used to determine UC entitlement"
+    reference = (
+        "https://www.legislation.gov.uk/uksi/2013/376/regulation/62/2021-04-06"
+    )
+    definition_period = YEAR
+
+    def formula(person, period, parameters):
+        return person("self_employment_income", period) > 0
+
+
+class UC_expected_hours_per_week(Variable):
```

Comment:
Do we have any other variables like this that are just parameters? Seems unnecessary and to set a burdensome precedent.

Reply:
I agree, but I wasn't sure yet of the exact circumstances for reduced expected hours of work, will revisit once it's clear

Reply:
Moved